{"DBCallId":"9593cd78-741f-49cf-8ce1-b9176d4bc9d7","TimeStamp":"2024-12-16T14:42:49.1612829-03:00","Query":" SELECT DATA_MOV_ABERTO , DATA_MOV_ABERTO FROM SEGUROS.SISTEMAS WHERE IDE_SISTEMA = ' ' WITH UR","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"b350382b-f1c8-4aaa-b801-fc8bb3a5e294","TimeStamp":"2024-12-16T14:42:57.310993-03:00","Query":" SELECT DATA_REFERENCIA , DATA_REFERENCIA FROM SEGUROS.RELATORIOS WHERE IDE_SISTEMA = 'PF' AND COD_RELATORIO = 'PF0617B' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"76e33ac5-519e-483a-9c22-c0e88260322e","TimeStamp":"2024-12-16T14:43:28.4738855-03:00","Query":" SELECT VALUE(MAX(NSAS_SIVPF) ,0) FROM SEGUROS.ARQUIVOS_SIVPF WHERE SIGLA_ARQUIVO = 'PRPSASSE' AND SISTEMA_ORIGEM = 4 WITH UR","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ExecuteReader requires an open connection. The connection's current state is Closed."}
{"DBCallId":"56a9aa6a-80bc-4127-bd02-68757dd98bbb","TimeStamp":"2024-12-16T14:43:37.8172713-03:00","Query":"SELECT A.NUM_BILHETE , A.NUM_APOLICE , A.FONTE , A.AGE_COBRANCA , A.NUM_MATRICULA , A.COD_AGENCIA , A.OPERACAO_CONTA , A.NUM_CONTA , A.DIG_CONTA , A.COD_CLIENTE , A.PROFISSAO , A.IDE_SEXO , A.ESTADO_CIVIL , A.OCORR_ENDERECO , A.COD_AGENCIA_DEB , A.OPERACAO_CONTA_DEB , A.NUM_CONTA_DEB , A.DIG_CONTA_DEB , A.OPC_COBERTURA , A.DATA_QUITACAO , A.VAL_RCAP , A.RAMO , A.DATA_VENDA , A.DATA_MOVIMENTO , A.NUM_APOL_ANTERIOR , A.SITUACAO , A.TIP_CANCELAMENTO , A.SIT_SINISTRO , A.COD_USUARIO , DATE(A.TIMESTAMP) , B.NUM_APOLICE , B.NUM_ENDOSSO , B.NUM_PROPOSTA , B.DATA_PROPOSTA , B.DATA_EMISSAO , B.NUM_RCAP , B.VAL_RCAP , B.DATA_INIVIGENCIA , B.DATA_TERVIGENCIA , B.COD_PRODUTO FROM SEGUROS.BILHETE A , SEGUROS.ENDOSSOS B WHERE A.SITUACAO = '9' AND B.NUM_APOLICE = A.NUM_APOLICE AND B.NUM_ENDOSSO = 0 AND B.DATA_EMISSAO >= '2024-07-19'","sqlCode":0,"rowsCount":0,"Time":"1 ms","Error":">>> Erro: ERROR [58005] [IBM][DB2.NET] SQL0902 An unexpected exception has occurred in Process: 13156 Thread 16 AppDomain: Name:testhost There are no context policies. Function: DB2DataReader.Read() CallStack: at System.Environment.get_StackTrace() at IBM.Data.Db2.DB2ConnPool.HandleUnknownErrors(String strFncMsg, Exception exception, Boolean bThrow) at IBM.Data.Db2.DB2DataReader.Read() at Dapper.SqlMapper.QueryImpl[T](IDbConnection cnn, CommandDefinition command, Type effectiveType)+MoveNext() in /_/Dapper/SqlMapper.cs:line 1201 at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection) at Dapper.SqlMapper.Query(IDbConnection cnn, String sql, Object param, IDbTransaction transaction, Boolean buffered, Nullable`1 commandTimeout, Nullable`1 commandType) in /_/Dapper/SqlMapper.cs:line 737 at IA_ConverterCommons.QueryBasis`1.Open(Object procParams) in E:\\PROJETOS\\FOURSYS\\SIAS\\SIAS-Batch\\IA_ConverterCommons\\Basis\\QueryBasis.cs:line 234 at Sias.PessoaFisica.DB2.PF0617B.PF0617B_C01_BILHETE.Open() in E:\\PROJETOS\\FOURSYS\\SIAS\\SIAS-Batch\\Sias.PessoaFisica\\DB2\\PF0617B\\PF0617B_C01_BILHETE.cs:line 73 at Code.PF0617B.R0050_00_SELECIONA_MOVTO_DB_OPEN_1() in E:\\PROJETOS\\FOURSYS\\SIAS\\SIAS-Batch\\Sias.PessoaFisica\\Code\\PF0617B.cs:line 1932 at FileTests.Test_DB.PF0617B_Tests_DB.PF0617B_Database() in E:\\PROJETOS\\FOURSYS\\SIAS\\SIAS-Batch\\FileTests.Test\\FileTests.Test\\PF0617B_Tests_DB.cs:line 27 at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor) at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr) at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters) at Xunit.Sdk.TestInvoker`1.CallTestMethod(Object testClassInstance) in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 151 at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_0.<<InvokeTestMethodAsync>b__1>d.MoveNext() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 278 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_0.<InvokeTestMethodAsync>b__1() at Xunit.Sdk.ExecutionTimer.AggregateAsync(Func`1 asyncAction) in /_/src/xunit.execution/Sdk/Frameworks/ExecutionTimer.cs:line 48 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.ExecutionTimer.AggregateAsync(Func`1 asyncAction) at Xunit.Sdk.TestInvoker`1.<>c__DisplayClass48_0.<InvokeTestMethodAsync>b__0() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 243 at Xunit.Sdk.ExceptionAggregator.RunAsync(Func`1 code) in /_/src/xunit.core/Sdk/ExceptionAggregator.cs:line 90 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.ExceptionAggregator.RunAsync(Func`1 code) at Xunit.Sdk.TestInvoker`1.InvokeTestMethodAsync(Object testClassInstance) in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 242 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestInvoker`1.InvokeTestMethodAsync(Object testClassInstance) at Xunit.Sdk.XunitTestInvoker.InvokeTestMethodAsync(Object testClassInstance) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestInvoker.cs:line 112 at Xunit.Sdk.TestInvoker`1.<RunAsync>b__47_0() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 207 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestInvoker`1.<RunAsync>b__47_0() at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) in /_/src/xunit.core/Sdk/ExceptionAggregator.cs:line 107 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) at Xunit.Sdk.TestInvoker`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestInvoker.cs:line 190 at Xunit.Sdk.XunitTestRunner.InvokeTestMethodAsync(ExceptionAggregator aggregator) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestRunner.cs:line 84 at Xunit.Sdk.XunitTestRunner.InvokeTestAsync(ExceptionAggregator aggregator) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestRunner.cs:line 67 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.XunitTestRunner.InvokeTestAsync(ExceptionAggregator aggregator) at Xunit.Sdk.TestRunner`1.<>c__DisplayClass43_0.<RunAsync>b__0() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestRunner.cs:line 149 at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) in /_/src/xunit.core/Sdk/ExceptionAggregator.cs:line 107 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.ExceptionAggregator.RunAsync[T](Func`1 code) at Xunit.Sdk.TestRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestRunner.cs:line 149 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestRunner`1.RunAsync() at Xunit.Sdk.XunitTestCaseRunner.RunTestAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestCaseRunner.cs:line 139 at Xunit.Sdk.TestCaseRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestCaseRunner.cs:line 82 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestCaseRunner`1.RunAsync() at Xunit.Sdk.XunitTestCase.RunAsync(IMessageSink diagnosticMessageSink, IMessageBus messageBus, Object[] constructorArguments, ExceptionAggregator aggregator, CancellationTokenSource cancellationTokenSource) in /_/src/xunit.execution/Sdk/Frameworks/XunitTestCase.cs:line 162 at Xunit.Sdk.XunitTestMethodRunner.RunTestCaseAsync(IXunitTestCase testCase) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestMethodRunner.cs:line 45 at Xunit.Sdk.TestMethodRunner`1.RunTestCasesAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestMethodRunner.cs:line 136 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestMethodRunner`1.RunTestCasesAsync() at Xunit.Sdk.TestMethodRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestMethodRunner.cs:line 106 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestMethodRunner`1.RunAsync() at Xunit.Sdk.XunitTestClassRunner.RunTestMethodAsync(ITestMethod testMethod, IReflectionMethodInfo method, IEnumerable`1 testCases, Object[] constructorArguments) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestClassRunner.cs:line 199 at Xunit.Sdk.TestClassRunner`1.RunTestMethodsAsync() at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestClassRunner`1.RunTestMethodsAsync() at Xunit.Sdk.TestClassRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestClassRunner.cs:line 175 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestClassRunner`1.RunAsync() at Xunit.Sdk.XunitTestCollectionRunner.RunTestClassAsync(ITestClass testClass, IReflectionTypeInfo class, IEnumerable`1 testCases) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestCollectionRunner.cs:line 185 at Xunit.Sdk.TestCollectionRunner`1.RunTestClassesAsync() at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestCollectionRunner`1.RunTestClassesAsync() at Xunit.Sdk.TestCollectionRunner`1.RunAsync() in /_/src/xunit.execution/Sdk/Frameworks/Runners/TestCollectionRunner.cs:line 101 at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine) at Xunit.Sdk.TestCollectionRunner`1.RunAsync() at Xunit.Sdk.XunitTestAssemblyRunner.RunTestCollectionAsync(IMessageBus messageBus, ITestCollection testCollection, IEnumerable`1 testCases, CancellationTokenSource cancellationTokenSource) in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestAssemblyRunner.cs:line 262 at Xunit.Sdk.XunitTestAssemblyRunner.<>c__DisplayClass14_2.<RunTestCollectionsAsync>b__2() in /_/src/xunit.execution/Sdk/Frameworks/Runners/XunitTestAssemblyRunner.cs:line 211 at System.Threading.Tasks.Task`1.InnerInvoke() at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread) at System.Threading.Tasks.Task.ExecuteEntry() at System.Threading.Tasks.SynchronizationContextTaskScheduler.<>c.<.cctor>b__8_0(Object s) at Xunit.Sdk.MaxConcurrencySyncContext.RunOnSyncContext(SendOrPostCallback callback, Object state) in /_/src/xunit.execution/Sdk/MaxConcurrencySyncContext.cs:line 106 at Xunit.Sdk.MaxConcurrencySyncContext.<>c__DisplayClass11_0.<WorkerThreadProc>b__0(Object _) in /_/src/xunit.execution/Sdk/MaxConcurrencySyncContext.cs:line 96 at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) at Xunit.Sdk.ExecutionContextHelper.Run(Object context, Action`1 action) in /_/src/xunit.execution/Sdk/Utility/ExecutionContextHelper.cs:line 110 at Xunit.Sdk.MaxConcurrencySyncContext.WorkerThreadProc() in /_/src/xunit.execution/Sdk/MaxConcurrencySyncContext.cs:line 96 at Xunit.Sdk.XunitWorkerThread.<>c.<QueueUserWorkItem>b__5_0(Object _) in /_/src/common/XunitWorkerThread.cs:line 37 at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state) at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread) InnerException Message: Year, Month, and Day parameters describe an un-representable DateTime. Check InnerException property for more detail. SQLSTATE=58005"}
{"DBCallId":"2255ba74-1b09-4185-8a47-aa6ca1e18614","TimeStamp":"2024-12-16T14:43:38.509848-03:00","Query":"SELECT COD_CLIENTE , COD_ENDERECO , OCORR_ENDERECO , ENDERECO , BAIRRO , CIDADE , SIGLA_UF , CEP , DDD , TELEFONE , FAX , TELEX , SIT_REGISTRO FROM SEGUROS.ENDERECOS WHERE COD_CLIENTE = '000000000' AND COD_ENDERECO = 1 ORDER BY OCORR_ENDERECO DESC","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"b3780766-d0cc-4920-bbcb-8e1fa25910d0","TimeStamp":"2024-12-16T14:43:54.6524927-03:00","Query":" SELECT SIT_PROPOSTA , CANAL_PROPOSTA , ORIGEM_PROPOSTA , COD_PRODUTO_SIVPF , NUM_IDENTIFICACAO , NUM_SICOB , COD_EMPRESA_SIVPF , NUM_PROPOSTA_SIVPF FROM SEGUROS.PROPOSTA_FIDELIZ WHERE NUM_SICOB = '000080000000017' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"2aea3298-b465-4c13-9fd1-777e2e7ad953","TimeStamp":"2024-12-16T14:43:55.288087-03:00","Query":" SELECT SIT_PROPOSTA FROM SEGUROS.HIST_PROP_FIDELIZ WHERE NUM_IDENTIFICACAO = '000000000000000' AND SIT_PROPOSTA = 'EMT' WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"a18cc64e-6cd1-4c87-a078-7ac86f93558f","TimeStamp":"2024-12-16T14:43:57.5439269-03:00","Query":" SELECT COD_CLIENTE , NOME_RAZAO , TIPO_PESSOA , CGCCPF , SIT_REGISTRO , DATA_NASCIMENTO FROM SEGUROS.CLIENTES WHERE COD_CLIENTE = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"1 ms","Error":""}
{"DBCallId":"af9ff771-65be-4d51-9185-b9b94d6ecaf6","TimeStamp":"2024-12-16T14:44:43.1759936-03:00","Query":" SELECT NUM_APOLICE , NUM_ENDOSSO , NUM_PROPOSTA , DATA_PROPOSTA , DATA_EMISSAO , NUM_RCAP , VAL_RCAP , DATA_INIVIGENCIA , DATA_TERVIGENCIA FROM SEGUROS.ENDOSSOS WHERE NUM_APOLICE = '0000000000000' AND NUM_ENDOSSO = '000000000' WITH UR","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ExecuteReader requires an open connection. The connection's current state is Closed."}
{"DBCallId":"9105c929-b840-4176-b254-c873fe428296","TimeStamp":"2024-12-16T14:44:44.3588713-03:00","Query":" SELECT COD_CLIENTE , COD_ENDERECO , OCORR_ENDERECO , ENDERECO , BAIRRO , CIDADE , SIGLA_UF , CEP , DDD , TELEFONE , FAX , TELEX , SIT_REGISTRO FROM SEGUROS.ENDERECOS WHERE COD_CLIENTE = '000000000' AND OCORR_ENDERECO = '0000' WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"c28b7e69-ca03-4009-8260-220b1f966e7c","TimeStamp":"2024-12-16T14:44:46.0233995-03:00","Query":"SELECT A.PRM_TARIFARIO_IX FROM SEGUROS.PARCELAS A WHERE A.NUM_APOLICE = '0000000000000' AND A.NUM_ENDOSSO = '000000000' ORDER BY TIMESTAMP","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"e566b6a4-0d4f-404c-a94e-78205ee4eddf","TimeStamp":"2024-12-16T14:45:16.4839439-03:00","Query":" SELECT COD_CBO , DESCR_CBO FROM SEGUROS.CBO WHERE DESCR_CBO = ' ' WITH UR","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ExecuteReader requires an open connection. The connection's current state is Closed."}
{"DBCallId":"dbdd942d-6fe4-4318-9571-b9b6acfbf1e0","TimeStamp":"2024-12-16T14:45:19.1936619-03:00","Query":" SELECT DISTINCT COD_EMPRESA_SIVPF , COD_PRODUTO_SIVPF , COD_RELAC FROM SEGUROS.PRODUTOS_SIVPF WHERE COD_EMPRESA_SIVPF = '0000' AND COD_PRODUTO_SIVPF = '0000' ORDER BY COD_EMPRESA_SIVPF , COD_PRODUTO_SIVPF , COD_RELAC WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"2fbc6f4c-2808-4179-abeb-4d48d54c8a72","TimeStamp":"2024-12-16T14:45:19.9661706-03:00","Query":" SELECT COD_FONTE , NUM_RCAP , NUM_PROPOSTA , VAL_RCAP , VAL_RCAP_PRINCIPAL , DATA_CADASTRAMENTO , DATA_MOVIMENTO , SIT_REGISTRO , COD_OPERACAO , NUM_TITULO FROM SEGUROS.RCAPS WHERE NUM_TITULO = '0000000000000' AND COD_OPERACAO BETWEEN 100 AND 199 WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"669ff808-ca69-467a-80cb-5b368ff1c822","TimeStamp":"2024-12-16T14:45:21.2037252-03:00","Query":" SELECT COD_FONTE , NUM_RCAP , NUM_PROPOSTA , VAL_RCAP , VAL_RCAP_PRINCIPAL , DATA_CADASTRAMENTO , DATA_MOVIMENTO , SIT_REGISTRO , COD_OPERACAO , NUM_TITULO FROM SEGUROS.RCAPS WHERE NUM_TITULO = '0000000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"673935dc-dce9-4bea-9dde-8eabd4331c6a","TimeStamp":"2024-12-16T14:45:42.2252681-03:00","Query":" SELECT A.PRM_TARIFARIO_IX FROM SEGUROS.PARCELAS A WHERE A.NUM_APOLICE = '0000000000000' AND A.NUM_PARCELA = '0000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"b49892b0-0c52-4ae9-bc3e-8af6b6259b36","TimeStamp":"2024-12-16T14:45:51.2635062-03:00","Query":"SELECT COD_CORRETOR , PCT_PART_CORRETOR , PCT_COM_CORRETOR FROM SEGUROS.APOLICE_CORRETOR WHERE NUM_APOLICE = '0000000000000'","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"ebef4fae-8fa3-4de1-b499-7d4edd4be543","TimeStamp":"2024-12-16T14:46:06.9890187-03:00","Query":"SELECT NUM_APOLICE , COD_SUBGRUPO , NUM_CERTIFICADO , DAC_CERTIFICADO , NUM_BENEFICIARIO , NOME_BENEFICIARIO , GRAU_PARENTESCO , PCT_PART_BENEFICIA , DATA_INIVIGENCIA , DATA_TERVIGENCIA FROM SEGUROS.BENEFICIARIOS WHERE NUM_CERTIFICADO = '000000000000000'","sqlCode":0,"rowsCount":14,"Time":"0 ms","Error":""}
{"DBCallId":"df1f3b3b-81b8-4c82-9959-76f36dd8bec4","TimeStamp":"2024-12-16T14:46:08.1669633-03:00","Query":" SELECT VAL_TARIFA FROM SEGUROS.TARIFA_BALCAO_CEF WHERE COD_EMPRESA = 0 AND NUM_MATRICULA = 9999999 AND TIPO_FUNCIONARIO = '5' AND NUM_CERTIFICADO = '000000000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"33886549-3999-4f53-857b-b37ba67a09a2","TimeStamp":"2024-12-16T14:46:09.1549363-03:00","Query":" SELECT NUM_PROPOSTA_SIVPF , NUM_SICOB FROM SEGUROS.CONVERSAO_SICOB WHERE NUM_SICOB = '000000000000000' WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"b440f262-cff2-4e3e-ba27-9550c7572281","TimeStamp":"2024-12-16T14:46:44.2385009-03:00","Query":" SELECT SIT_PROPOSTA , CANAL_PROPOSTA , ORIGEM_PROPOSTA , COD_PRODUTO_SIVPF , NUM_IDENTIFICACAO , NUM_SICOB FROM SEGUROS.PROPOSTA_FIDELIZ WHERE NUM_PROPOSTA_SIVPF = '000080021058960' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"be4f1b36-57cc-4609-af11-f6e7d9ae7728","TimeStamp":"2024-12-16T14:46:45.0255321-03:00","Query":" UPDATE SEGUROS.PROPOSTA_FIDELIZ SET SITUACAO_ENVIO = 'S' , COD_USUARIO = 'PF0617B' , SIT_PROPOSTA = 'EMT' WHERE NUM_SICOB = '0080000000017'","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"c9a8acb5-2df9-41e7-9e01-64db2feb61ca","TimeStamp":"2024-12-16T14:46:45.7924307-03:00","Query":" SELECT NUM_APOL_ANTERIOR FROM SEGUROS.BILHETE WHERE NUM_BILHETE = '0080000000017' WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"bdad12a2-8083-4a14-8766-e1bf8d418455","TimeStamp":"2024-12-16T14:46:46.5026085-03:00","Query":" SELECT NUM_PROPOSTA_SIVPF , SIT_PROPOSTA FROM SEGUROS.PROPOSTA_FIDELIZ WHERE NUM_SICOB = '0080000000017'","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"c40c86e4-da3e-4527-b3ea-9fffc9b9190a","TimeStamp":"2024-12-16T14:46:48.0885536-03:00","Query":" SELECT COD_CLIENTE , COD_IDENTIFICACAO , NOM_ORGAO_EXP , DTH_EXPEDICAO , COD_UF FROM SEGUROS.GE_DOC_CLIENTE WHERE COD_CLIENTE = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"5cd087c4-70da-44de-acd6-e0708c4f860e","TimeStamp":"2024-12-16T14:46:48.7538283-03:00","Query":"SELECT C.DATA_EMISSAO FROM SEGUROS.BILHETE A , SEGUROS.ENDOSSOS C WHERE A.RAMO IN (81 ,82 ,37) AND A.SITUACAO = '9' AND C.NUM_APOLICE = A.NUM_APOLICE AND C.NUM_ENDOSSO = 0 AND C.DATA_EMISSAO < '2020-01-01' AND C.DATA_EMISSAO > '2021-12-12' AND NOT EXISTS (SELECT B.NUM_SICOB FROM SEGUROS.PROPOSTA_FIDELIZ B WHERE A.NUM_BILHETE = B.NUM_SICOB) ORDER BY C.DATA_EMISSAO ASC","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"a5ad17e4-09a1-4536-a560-8f9978b7eba1","TimeStamp":"2024-12-16T14:46:51.8087509-03:00","Query":" SELECT NUM_DV_LOTERICO FROM FDRCAP.FC_LOTERICO WHERE NUM_LOTERICO = '000000000' FETCH FIRST 1 ROWS ONLY WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [42501] [IBM][DB2] SQL0551N The statement failed because the authorization ID does not have the required authorization or privilege to perform the operation. Authorization ID: \"SISSMZ1H\". Operation: \"SELECT\". Object: \"FDRCAP.FC_LOTERICO\"."}
{"DBCallId":"9189b7d6-fd2e-4e3a-b1e9-0b4652b134eb","TimeStamp":"2024-12-16T14:46:52.5664642-03:00","Query":" SELECT NUM_DV_LOTERICO FROM FDRCAP.FC_LOTERICO WHERE NUM_LOTERICO = '000000000' FETCH FIRST 1 ROWS ONLY WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [42501] [IBM][DB2] SQL0551N The statement failed because the authorization ID does not have the required authorization or privilege to perform the operation. Authorization ID: \"SISSMZ1H\". Operation: \"SELECT\". Object: \"FDRCAP.FC_LOTERICO\"."}
{"DBCallId":"145eb6a1-efcd-4d87-b14e-808edf25f706","TimeStamp":"2024-12-16T14:47:07.7726759-03:00","Query":" SELECT NUM_DV_LOTERICO FROM FDRCAP.FC_LOTERICO WHERE NUM_LOTERICO = '000000000' FETCH FIRST 1 ROWS ONLY WITH UR","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [42501] [IBM][DB2] SQL0551N The statement failed because the authorization ID does not have the required authorization or privilege to perform the operation. Authorization ID: \"SISSMZ1H\". Operation: \"SELECT\". Object: \"FDRCAP.FC_LOTERICO\"."}
{"DBCallId":"64e22782-333a-41d4-af7d-e9ddc2618a7d","TimeStamp":"2024-12-16T14:47:08.6820896-03:00","Query":" SELECT NUM_DV_LOTERICO FROM FDRCAP.FC_LOTERICO WHERE NUM_LOTERICO = '000000000' FETCH FIRST 1 ROWS ONLY WITH UR","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [42501] [IBM][DB2] SQL0551N The statement failed because the authorization ID does not have the required authorization or privilege to perform the operation. Authorization ID: \"SISSMZ1H\". Operation: \"SELECT\". Object: \"FDRCAP.FC_LOTERICO\"."}
{"DBCallId":"855eb2ac-979d-4a4e-a447-46162afd88c6","TimeStamp":"2024-12-16T14:47:09.5163296-03:00","Query":" SELECT NUM_DV_LOTERICO FROM FDRCAP.FC_LOTERICO WHERE NUM_LOTERICO = '000000000' FETCH FIRST 1 ROWS ONLY WITH UR","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [42501] [IBM][DB2] SQL0551N The statement failed because the authorization ID does not have the required authorization or privilege to perform the operation. Authorization ID: \"SISSMZ1H\". Operation: \"SELECT\". Object: \"FDRCAP.FC_LOTERICO\"."}
{"DBCallId":"1799a3ce-134c-4304-9fdd-63f46e64134c","TimeStamp":"2024-12-16T14:47:10.3777529-03:00","Query":" SELECT COD_HIERARQUIA FROM SEGUROS.PRODUTORES WHERE NUM_MATRICULA = '000000000000000' AND TIPO_PRODUTOR = '3' FETCH FIRST 1 ROWS ONLY WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"521f0048-1705-4144-93f8-df2f248c542c","TimeStamp":"2024-12-16T14:47:11.1234952-03:00","Query":" SELECT COD_HIERARQUIA FROM SEGUROS.PRODUTORES WHERE NUM_MATRICULA = '000000000000000' FETCH FIRST 1 ROWS ONLY WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"09c565e2-56a3-441e-a657-e5812e1c79c2","TimeStamp":"2024-12-16T14:47:12.2530032-03:00","Query":" INSERT INTO SEGUROS.ARQUIVOS_SIVPF VALUES (' ', '0000', '000000000', '2021-01-01', '000000000', '2022-01-01')","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23503] [IBM][DB2] SQL0530N The insert or update value of the FOREIGN KEY \"FK00004\" is not equal to any value of the parent key of the parent table."}
{"DBCallId":"f1d18e3a-956f-4db1-8c14-a81655346ef1","TimeStamp":"2024-12-16T14:47:12.8465167-03:00","Query":" SELECT DIA_SEMANA FROM SEGUROS.CALENDARIO WHERE DATA_CALENDARIO = '2021-01-01' WITH UR","sqlCode":0,"rowsCount":1,"Time":"1 ms","Error":""}
{"DBCallId":"1481945d-f3d0-4abc-8866-1d5dabb7f8ce","TimeStamp":"2024-12-16T14:47:43.9775046-03:00","Query":" UPDATE SEGUROS.RELATORIOS SET DATA_REFERENCIA = '2024-07-19', TIMESTAMP = CURRENT TIMESTAMP WHERE IDE_SISTEMA = 'PF' AND COD_RELATORIO = 'PF0617B'","sqlCode":999,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"e5535f25-63af-4152-b87f-eb20f9913fe3","TimeStamp":"2024-12-16T14:47:47.0107906-03:00","Query":"SELECT COD_PESSOA , SEQ_EMAIL , EMAIL , SITUACAO_EMAIL FROM SEGUROS.PESSOA_EMAIL WHERE COD_PESSOA = '000000000' ORDER BY SEQ_EMAIL","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"b7f963d1-5fd7-4760-9524-2c4a8c38e616","TimeStamp":"2024-12-16T14:47:47.8003632-03:00","Query":" UPDATE SEGUROS.PROPOSTA_FIDELIZ SET SITUACAO_ENVIO = 'S' WHERE NUM_SICOB = '000080000000017'","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"f8680fac-5344-4810-9bca-4c1a7a200197","TimeStamp":"2024-12-16T14:47:48.4675322-03:00","Query":" UPDATE SEGUROS.PROPOSTA_FIDELIZ SET SITUACAO_ENVIO = 'S' , SIT_PROPOSTA = 'EMT' WHERE NUM_PROPOSTA_SIVPF = '000080021058960'","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"8dfb8421-5ce7-4fab-af2f-af10321a08e1","TimeStamp":"2024-12-16T14:47:49.2376034-03:00","Query":" SELECT COD_PESSOA , CPF , DATA_NASCIMENTO , SEXO , TIPO_IDENT_SIVPF , NUM_IDENTIDADE , ORGAO_EXPEDIDOR , UF_EXPEDIDORA , DATA_EXPEDICAO , COD_CBO , COD_USUARIO , ESTADO_CIVIL , TIMESTAMP FROM SEGUROS.PESSOA_FISICA WHERE CPF = '00000000000' AND DATA_NASCIMENTO = '2021-02-02' AND SEXO = ' ' WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"51966bd4-2769-4f71-8044-6d350505cd61","TimeStamp":"2024-12-16T14:47:52.363205-03:00","Query":" SELECT COD_PESSOA , CGC , NOME_FANTASIA , COD_USUARIO , TIMESTAMP FROM SEGUROS.PESSOA_JURIDICA WHERE CGC = '000000000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"95150849-fde4-41ac-bf9c-7be7693e5c7f","TimeStamp":"2024-12-16T14:47:53.0653698-03:00","Query":" INSERT INTO SEGUROS.PESSOA VALUES ('000000000', ' ', CURRENT TIMESTAMP, ' ', ' ', NULL)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET121\" constrains table \"0000008578\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"127b698c-898e-4ee8-9b73-d597fded65d9","TimeStamp":"2024-12-16T14:47:54.1017186-03:00","Query":" SELECT VALUE(MAX(COD_PESSOA) ,0) FROM SEGUROS.PESSOA WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"43b96ae4-7e1c-40cf-961d-0c19e64a7042","TimeStamp":"2024-12-16T14:47:54.8381669-03:00","Query":" INSERT INTO SEGUROS.PESSOA_FISICA VALUES ('000000000', '00000000000', '2000-01-01', ' ', ' ', ' ', CURRENT TIMESTAMP, ' ', ' ', ' ', '2021-02-02', '000000000', ' ')","sqlCode":999,"rowsCount":0,"Time":"1 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET132\" constrains table \"0000008270\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"b2862bfe-c48c-49ee-891b-d547383cbbc4","TimeStamp":"2024-12-16T14:47:55.6274179-03:00","Query":" INSERT INTO SEGUROS.PESSOA_FISICA VALUES ('000000000', '00000000000', '2000-01-01', ' ', ' ', ' ', CURRENT TIMESTAMP, ' ', ' ', ' ', '2021-02-02', '000000000', ' ')","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET132\" constrains table \"0000008270\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"ca4b3df4-1909-49cc-9561-300f0c105859","TimeStamp":"2024-12-16T14:47:56.4727319-03:00","Query":" INSERT INTO SEGUROS.PESSOA_JURIDICA VALUES ('001128805', '000000000000000', 'ELIANE FERREIRA ', 'PF0617B ', CURRENT TIMESTAMP)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET133\" constrains table \"0000015E09\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"48719d03-f741-438c-9e7b-2a531affdf54","TimeStamp":"2024-12-16T14:47:57.2207007-03:00","Query":"SELECT OCORR_ENDERECO , COD_PESSOA , ENDERECO , TIPO_ENDER , BAIRRO , CEP , CIDADE , SIGLA_UF , SITUACAO_ENDERECO FROM SEGUROS.PESSOA_ENDERECO WHERE COD_PESSOA = '000000000' ORDER BY OCORR_ENDERECO","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"7a4463cf-5a9e-4dac-b5df-c97842f643d2","TimeStamp":"2024-12-16T14:47:57.9428644-03:00","Query":" SELECT VALUE(MAX(SEQ_EMAIL) ,0) FROM SEGUROS.PESSOA_EMAIL WHERE COD_PESSOA = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"c444c2b2-9e97-4680-a735-1e43b481776d","TimeStamp":"2024-12-16T14:47:59.6335976-03:00","Query":" INSERT INTO SEGUROS.PESSOA_EMAIL VALUES ('000000000', '0000', ' ', ' ', ' ', CURRENT TIMESTAMP)","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET128\" constrains table \"000003A768\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"d90d37a1-6f1e-4e13-b529-03d0def5312d","TimeStamp":"2024-12-16T14:48:00.3156881-03:00","Query":" SELECT COD_PESSOA , NOME_PESSOA , TIPO_PESSOA , TIMESTAMP , COD_USUARIO FROM SEGUROS.PESSOA WHERE COD_PESSOA = '042584833' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"3ead0fbe-c9aa-48a3-acae-d19303bde2d3","TimeStamp":"2024-12-16T14:48:00.9043343-03:00","Query":" SELECT VALUE(MAX(SEQ_EMAIL) ,0) FROM SEGUROS.PESSOA_EMAIL WHERE COD_PESSOA = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"bd8dcfa9-0bcb-4a74-9997-0b12f8059637","TimeStamp":"2024-12-16T14:48:01.5355302-03:00","Query":" SELECT COD_PESSOA , SEQ_EMAIL , EMAIL , SITUACAO_EMAIL , COD_USUARIO , TIMESTAMP FROM SEGUROS.PESSOA_EMAIL WHERE COD_PESSOA = '000000000' AND SEQ_EMAIL = '0000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"fe749967-1e01-4a0a-911a-c8a50a451c59","TimeStamp":"2024-12-16T14:48:02.0987425-03:00","Query":" SELECT VALUE(MAX(OCORR_ENDERECO) ,0) FROM SEGUROS.PESSOA_ENDERECO WHERE COD_PESSOA = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"bc13fee8-50db-4196-b118-d14c68777e28","TimeStamp":"2024-12-16T14:48:02.5277793-03:00","Query":" INSERT INTO SEGUROS.PESSOA_ENDERECO VALUES ('000000000', '0000', ' ', '0000', NULL, ' ', '000000000', ' ', ' ', ' ', ' ', CURRENT TIMESTAMP)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23503] [IBM][DB2] SQL0530N The insert or update value of the FOREIGN KEY \"FK00028\" is not equal to any value of the parent key of the parent table."}
{"DBCallId":"c90af734-629a-422b-9cdc-f2b2e7cab20f","TimeStamp":"2024-12-16T14:48:02.8839069-03:00","Query":" SELECT SITUACAO_FONE FROM SEGUROS.PESSOA_TELEFONE WHERE COD_PESSOA = '000000000' AND TIPO_FONE = '0000' AND NUM_FONE = '00000000000' AND DDD = '0000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"40eee757-ff08-47f6-b7e8-9202dd25c449","TimeStamp":"2024-12-16T14:48:03.1983708-03:00","Query":" SELECT VALUE(MAX(SEQ_FONE) ,0) FROM SEGUROS.PESSOA_TELEFONE WHERE COD_PESSOA = '000000000' WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"2cf84a5e-d9af-4481-88f1-49d045d865b8","TimeStamp":"2024-12-16T14:48:03.5551346-03:00","Query":" INSERT INTO SEGUROS.PESSOA_TELEFONE VALUES ('000000000', '0000', '0000', '0000', '0000', '00000000000', ' ', ' ', CURRENT TIMESTAMP)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"IPGET134\" constrains table \"000000827F\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"ee8b22b1-7007-47c0-8a89-563e33990fe7","TimeStamp":"2024-12-16T14:48:03.8305448-03:00","Query":" SELECT DISTINCT COD_PRODUTO_SIVPF , COD_RELAC FROM SEGUROS.PRODUTOS_SIVPF WHERE COD_EMPRESA_SIVPF = '0000' AND COD_PRODUTO_SIVPF = '0000' ORDER BY COD_PRODUTO_SIVPF , COD_RELAC WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"bb48d55f-f0a9-48d2-b6c1-34e633d9a292","TimeStamp":"2024-12-16T14:48:04.1275066-03:00","Query":" SELECT COD_PESSOA , COD_RELAC FROM SEGUROS.R_PESSOA_TIPORELAC WHERE COD_PESSOA = '000000000' AND COD_RELAC = '0000' WITH UR","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":""}
{"DBCallId":"265c286a-833c-4889-9b3e-49cac4e1b1d7","TimeStamp":"2024-12-16T14:48:04.3988512-03:00","Query":" INSERT INTO SEGUROS.R_PESSOA_TIPORELAC VALUES ('000000000', '0000')","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23503] [IBM][DB2] SQL0530N The insert or update value of the FOREIGN KEY \"FK00002\" is not equal to any value of the parent key of the parent table."}
{"DBCallId":"e6f4ed7e-ef5f-4cca-bc49-1bc8efb70097","TimeStamp":"2024-12-16T14:48:04.5970971-03:00","Query":" INSERT INTO SEGUROS.IDENTIFICA_RELAC VALUES ('000000000000000', '000000000', '0000', ' ', CURRENT TIMESTAMP)","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23503] [IBM][DB2] SQL0530N The insert or update value of the FOREIGN KEY \"FK00003\" is not equal to any value of the parent key of the parent table."}
{"DBCallId":"87d64049-2fda-4135-b517-41d87ec64820","TimeStamp":"2024-12-16T14:48:04.9307054-03:00","Query":" SELECT VALUE(MAX(NUM_IDENTIFICACAO) ,0) FROM SEGUROS.IDENTIFICA_RELAC WITH UR","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}
{"DBCallId":"31a7e12d-04eb-4f33-9199-4465cec00385","TimeStamp":"2024-12-16T14:48:05.3248127-03:00","Query":" INSERT INTO SEGUROS.PROPOSTA_FIDELIZ (NUM_PROPOSTA_SIVPF ,NUM_IDENTIFICACAO ,COD_EMPRESA_SIVPF ,COD_PESSOA ,NUM_SICOB ,DATA_PROPOSTA ,COD_PRODUTO_SIVPF ,AGECOBR ,DIA_DEBITO ,OPCAOPAG ,AGECTADEB ,OPRCTADEB ,NUMCTADEB ,DIGCTADEB ,PERC_DESCONTO ,NRMATRVEN ,AGECTAVEN ,OPRCTAVEN ,NUMCTAVEN ,DIGCTAVEN ,CGC_CONVENENTE ,NOME_CONVENENTE ,NRMATRCON ,DTQITBCO ,VAL_PAGO ,AGEPGTO ,VAL_TARIFA ,VAL_IOF ,DATA_CREDITO ,VAL_COMISSAO ,SIT_PROPOSTA ,COD_USUARIO ,CANAL_PROPOSTA ,NSAS_SIVPF ,ORIGEM_PROPOSTA ,TIMESTAMP ,NSL ,NSAC_SIVPF ,SITUACAO_ENVIO ,OPCAO_COBER ,COD_PLANO ,NOME_CONJUGE ,DATA_NASC_CONJUGE ,PROFISSAO_CONJUGE ,FAIXA_RENDA_IND ,FAIXA_RENDA_FAM ,IND_TP_PROPOSTA ,IND_TIPO_CONTA) VALUES ('000080021058960' , '000000000000001' , '0003' , '000000000' , '000080000000017' , '2022-01-02' , '0080' , '0000' , '0000' , ' ' , '0000' , '0000' , '0000000000000' , '0000' , '000.00' , '000000000000000' , '0000' , '0000' , '0000000000000' , '0000' , '00000000000000' , ' ' , '000000000' , '2021-02-02' , '0000000000000.00' , '0000' , '0000000000000.00' , '0000.00' , '2023-01-05' , '0000000000000.00' , 'EMT' , ' ' , '0001' , '000000000' , '0006' , CURRENT TIMESTAMP , '000000000' , '000000000' , ' ' , ' ' , '0000' , ' ' , NULL , '000000000' , ' ' , ' ' , NULL, ' ')","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23513] [IBM][DB2] SQL0545N The requested operation is not allowed because a row does not satisfy the check constraint \"PF002C2\"."}
{"DBCallId":"055ff9c1-e496-40af-ac74-8931ad936111","TimeStamp":"2024-12-16T14:48:05.7185949-03:00","Query":" INSERT INTO SEGUROS.PROP_SASSE_BILHETE VALUES ('000000000000000' , ' ' , ' ' , CURRENT TIMESTAMP , '2022-01-01' , 0 )","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"PF011IP\" constrains table \"0000008270\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"168b443e-2841-4b56-9965-6049d9be549a","TimeStamp":"2024-12-16T14:48:06.0730688-03:00","Query":" INSERT INTO SEGUROS.HIST_PROP_FIDELIZ VALUES ( '000000000000000' , '2023-02-05' , '000000000' , '000000000' , ' ' , ' ' , '000000000' , '0000' , '0000')","sqlCode":0,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [42802] [IBM][DB2] SQL0117N The number of values assigned is not the same as the number of specified or implied columns or variables."}
{"DBCallId":"d9a9b263-8905-4a35-9feb-c503df4e8faf","TimeStamp":"2024-12-16T14:48:06.3999828-03:00","Query":" INSERT INTO SEGUROS.BENEF_PROP_AZUL VALUES ('0000000000000', '0000', '000000000', '0000', '0000', '0000', ' ', ' ', '000.00', '2000-01-01')","sqlCode":999,"rowsCount":0,"Time":"0 ms","Error":">>> Erro: ERROR [23505] [IBM][DB2] SQL0803N One or more values in the INSERT statement, UPDATE statement, or foreign key update caused by a DELETE statement are not valid because the primary key, unique constraint or unique index identified by \"I1VAT066\" constrains table \"0000008278\" from having duplicate values for the index key. SQLSTATE=23505"}
{"DBCallId":"fc326852-ce6a-47a6-8bd8-7f836ba13b3c","TimeStamp":"2024-12-16T14:48:06.6364525-03:00","Query":" SELECT VALUE(MAX(NUM_BENEFICIARIO) ,0) FROM SEGUROS.BENEF_PROP_AZUL WHERE NUM_PROPOSTA_AZUL = '0000000000000'","sqlCode":0,"rowsCount":1,"Time":"0 ms","Error":""}